OTKit/README.md

```diff
@@ -11,15 +11,21 @@
 
 ## Usage
 
-Add the needed tokens to your devDependnecies:
+Run the command below will add the needed tokens to your project:
 
-```
+```bash
 $ npm install save-dev <token-name>
```

Comment:
I think the correct flag should be `--save-dev` thats why we had that little issue earlier when we tried it.